ironfish/src/wallet/wallet.ts

```diff
@@ -991,6 +992,22 @@ export class Wallet {
       )
     }
 
+    if (options.outputs) {
+      const errorMemos: number[] = []
```

Comment:
This looks fine, but I personally wouldn't accumulate errors. One error is enough to make the caller realize they aren't validating the input of byte lengths which is a fix they need to make in their code.

Reply:
I am fine with that route too, will allow us to output message too